docs/sources/dashboards/time-range-controls.md

```diff
@@ -103,6 +103,8 @@ To access the dashboard time settings, click the **Dashboard settings** (gear) i
   - Standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), including UTC.
 - **Auto-refresh -** Customize the options displayed for relative time and the auto-refresh options. Entries are comma separated and accept any valid time unit.
 - **Now delay now- -** Override the `now` value by entering a time delay. Most commonly, this feature is used to accommodate known delays in data aggregation to avoid null values.
+- **Max time back -** Prevents the users from setting the dashboard `from time` to a value past the max time back. By default, the max time back is not set. The input should be the relative time back value, followed by a unit suffix (ms, s, m, h, d, w, M, y), e.g. `1y`. This will be interpreted as `now-1y` by Grafana.
```

Comment:
`Max time back` doesn't seem that user friendly, WDTH @oddlittlebird?

Reply:
Agreed. I thought I was getting time back at first glance. 

Maybe something like "Limit time range"? We need to make clear that we are limiting the ability of users to query more than XX amount of time in the past, or restricting the value they can inter in the From field.

Reply:
I think "Time [range] lower limit" would be the most accurate description, but I also think using either the word "past" or "back" makes it more clear to the user how it is calculated/what kind of input is expected (i.e. how far back from now a user can go). Any other suggestions?

Reply:
"Time [range] lower limit" is also not clear to me. 

This is also not clear: "Prevents the users from setting the dashboard `from time` to a value past the max time back." What does `from time` mean?

I will try to offer a better wording suggestion.

Reply:
Lower limit and `from time` are both attempts to reference the start of the time range, and [range] meant to suggest that the word range could be included or excluded in that phrase in actual use to clear any confusion. How about "time range start" and "time range end" for all references to the endpoints of the time range?

Reply:
I like this idea. Thank you for the clarification.

Reply:
On a separate note, can we revise the "**Now delay now- -**" phrase? Maybe "Add delay to now"?

Reply:
Maybe out of scope for this PR, it matches the setting name and should be a noun. I think it is trying to do the same thing, which is to clarify that the setting is relative to 'now'. As for the original concern with the setting name, does "Time range start limit" seem like a reasonable alternative to "Max time back"? And then in the tooltip/docs clarify that the actual point in time is calculated as "now - \<setting>"?

Reply:
"Time range start limit" makes sense, let us go ahead with it.